docs/src/main/asciidoc/grpc-getting-started.adoc

```diff
@@ -0,0 +1,318 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Getting Started with Quarkus gRPC
+
+include::./attributes.adoc[]
+
+This page explains how to start using Quarkus gRPC.
+While this page describes how to configure it with Maven, it is also possible to use Gradle.
+
+Let's imagine you have a regular Quarkus project, generated from the https://code.quarkus.io[Quarkus project generator].
+The default configuration is enough, but you can also select some extensions if you want.
+
+== Configuring your project
+
+Edit the `pom.xml` file to add the quarkus gRPC extension dependency and the JSR 305 (just under `<dependencies>`):
+
+[source, xml]
+----
+<dependency>
+  <groupId>io.quarkus</groupId>
+  <artifactId>quarkus-grpc</artifactId>
+</dependency>
+<dependency>
+ <groupId>com.google.code.findbugs</groupId>
+ <artifactId>jsr305</artifactId>
+ <scope>provided</scope>
```

Comment:
This is annoying because you can't just add the extension.

Reply:
It's an ongoing discussion with @michalszynkiewicz . I hope that we would be able to remove it from there. Because I agree... should not be required.

Reply:
I agree that requiring users to add a dependency is not the way to go.
Let's drop the ban for jsr305 (at least for now)

Reply:
Fixed.